IP address parsing failures from invalid SNMP data

## Error

```
[ERROR] Failed to parse IP: .0/2 is not a valid ipv4 address
[ERROR] Failed to parse IP: Could not parse into IP: 000aa00a1900
```

## Details

- **Severity:** ERROR

## Description

IP address parsing fails when SNMP returns malformed or unexpected data:
- `.0/2` — not a valid IPv4 address (likely truncated or corrupted SNMP response)
- `000aa00a1900` — hex string that cannot be parsed as an IP address

These errors indicate missing input validation before attempting to parse SNMP-sourced values as IP addresses.

## Suggested Fix

Add input validation to check for valid IP address format before parsing. Skip or log a warning for entries with clearly invalid data rather than throwing an error.

--- a/LibreNMS/Modules/Ipv4Addresses.php
+++ b/LibreNMS/Modules/Ipv4Addresses.php
@@ -199,6 +199,10 @@ class Ipv4Addresses implements Module
                 $entAddr = $data['ipAdEntAddr'] ?? '';
                 $addr = (preg_match('/(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})$/', (string) $entAddr, $tmp)) ? $entAddr : $ipAddr;
 
+                if (! IPv4::isValid($addr)) {
+                    return null;
+                }
+
                 return new Ipv4Address([
                     'port_id' => PortCache::getIdFromIfIndex($data['ipAdEntIfIndex'] ?? 0, $device),
                     'ipv4_address' => $addr,

